### PR TITLE
feat: support Kratos cleanup sql cronjob

### PR DESCRIPTION
**Pull request by David-Wobrock**
Add support for a cronjob to run https://www.ory.sh/docs/kratos/cli/kratos-cleanup-sql

Based a lot on the Hydra janitor cronjob.

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [x] I have referenced an issue containing the design document if my change introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: helm/charts/kratos/values.yaml [hunk 1]
```diff
@@ -560,6 +560,80 @@ watcher:
...
+    # -- Configure the containers' SecurityContext for the cleanup cronjob
+    securityContext:
+      capabilities:
+        drop:
+          - ALL
+      readOnlyRootFilesystem: true
+      runAsNonRoot: true
+      runAsUser: 100
+      allowPrivilegeEscalation: false
+      privileged: false
```
Comment (Demonsthere) [inline helm/charts/kratos/values.yaml:618]:
```suggestion
      privileged: false
      seccompProfile:
        type: RuntimeDefault
```

File: helm/charts/kratos/values.yaml [hunk 2]
```diff
@@ -560,6 +560,80 @@ watcher:
   # -- Label key used for managing applications
   watchLabelKey: "ory.sh/watcher"
 
+## -- SQL cleanup cron job configuration
```
Comment (Demonsthere) [inline helm/charts/kratos/values.yaml:563]: Please keep only one `#` as that tend to confuse the helm-docs automation :)

**Review by Demonsthere — COMMENTED** [review]
Hello there :) 
Good job! Just leaving some minor remarks, but the whole structure looks good!

**Comment by David-Wobrock** [comment]
Hi @Demonsthere  👋 

> Hello there :) Good job! Just leaving some minor remarks, but the whole structure looks good!

Thanks for review 😄 I pushed some edits.

**Review by Demonsthere — APPROVED** [review]
LGTM! :)

**Comment by David-Wobrock** [comment]
Any idea when a new release will be made? 😇

**Pull request merged**